### ISSUE TITLE
'Stationary distance has NA values' in coarse-grained transition matrix

### ISSUE THREAD

**Issue by sneddonucsf** [theislab/cellrank#946]
Hello, I am trying to plot the coarse-grained transition matrix of my data computed with the Pseudotime Kernel and am running an error where it makes most of the plot, but the stationary distribution doesn't population on the plot and I get the following error: 

```python
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/var/folders/rx/h1kgwvvd4td05bp8pxvzb42r0000gn/T/ipykernel_6340/3594035503.py in <module>
----> 1 g_slng_soft.plot_coarse_T(text_kwargs={"fontsize": 10})

~/opt/miniconda3/envs/my_env/lib/python3.7/site-packages/cellrank/tl/estimators/terminal_states/_gpcca.py in plot_coarse_T(self, show_stationary_dist, show_initial_dist, cmap, xtick_rotation, annotate, show_cbar, title, figsize, dpi, save, text_kwargs, **kwargs)
    650             annotate_heatmap(im)
    651             if show_stationary_dist:
--> 652                 annotate_dist_ax(stat_ax, coarse_stat_d.values)
    653             if show_initial_dist:
    654                 annotate_dist_ax(init_ax, coarse_init_d.values)

~/opt/miniconda3/envs/my_env/lib/python3.7/site-packages/cellrank/tl/estimators/terminal_states/_gpcca.py in annotate_dist_ax(ax, data, valfmt)
    527 
    528             for i, val in enumerate(data):
--> 529                 kw.update(color=_get_black_or_white(im.norm(val), cmap))
    530                 ax.text(
    531                     i,

~/opt/miniconda3/envs/my_env/lib/python3.7/site-packages/cellrank/tl/_colors.py in _get_black_or_white(value, cmap)
    138 def _get_black_or_white(value: float, cmap) -> str:
    139     if not (0.0 <= value <= 1.0):
--> 140         raise ValueError(f"Value must be in range `[0, 1]`, found `{value}`.")
    141 
    142     r, g, b, *_ = (int(c * 255) for c in cmap(value))

ValueError: Value must be in range `[0, 1]`, found `nan`.```
```
![image](https://user-images.githubusercontent.com/12053091/189801019-d79814e9-8112-4fed-bfd2-c8a05291843f.png)

I am able to go through all other parts of the kernel. My data contains 4 macrostates, each of which I am designating as a terminal state. Here is what I ran up to the coarse-grained transition matrix: 

```
from cellrank.tl.kernels import PseudotimeKernel
from cellrank.tl.estimators import GPCCA
pk_slng_soft = cr.tl.kernels.PseudotimeKernel(adata, time_key='Slingshot_avg')
pk_slng_soft.compute_transition_matrix(threshold_scheme='soft')
pk_slng_soft.compute_projection(basis="umap")
g_slng_soft = GPCCA(pk_slng_soft)
g_slng_soft.compute_schur(n_components=30)
g_slng_soft.compute_macrostates(n_states = 4, 
                                cluster_key="annotation", 
                                n_cells = 100)
g_slng_soft.plot_macrostates(discrete= True, 
                             size=50, 
                             basis="umap", 
                            legend_loc = None, 
                            title = "")
```
![image](https://user-images.githubusercontent.com/12053091/189800685-ed1dbb94-b212-458d-9b5b-b8a5e6c7d3a4.png)

```
g_slng_soft.set_terminal_states_from_macrostates(["Alpha", "Beta", "Epsilon", "Delta"])
g_slng_soft.plot_coarse_T()
```
where I then get the error. My kernel/estimator: 
```
GPCCA[n=14231, kernel=<PseudotimeKernel>]
```

Thank you! 

#### Versions:
```
cellrank==1.5.1 scanpy==1.9.1 anndata==0.8.0 numpy==1.21.6 numba==0.55.2 scipy==1.7.3 pandas==1.3.5 pygpcca==1.0.3 scikit-learn==1.0.2 statsmodels==0.13.2 python-igraph==0.9.11 scvelo==0.2.4 pygam==0.8.0 matplotlib==3.5.3 seaborn==0.12.0
```

**Comment by michalk8** [comment]
Hi @sneddonucsf , sorry for the slow reply, will look into this.

**Comment by sneddonucsf** [comment]
No worries @michalk8, thank you!

**Comment by michalk8** [comment]
Should be fixed n #951 , feel free to re-open if it's still an issue.
